### ISSUE TITLE
sysimage loading - check Manifest.toml SHA

### ISSUE THREAD

**Issue by phlavenk** [julia-vscode/julia-vscode#1352]
I love the ease of use of statically recompiling the sysimage for a project.
What annoys me is that the starting mechanism for julia process relies on file dates instead of the SHA of Manifest.toml and Project.toml.  The current implemetation forces me to build new sysimage everytime I do any git operation, typically stash/pop of project/manifes files.
Do I understand correctly this startup mechanism is currently coded in the TypeScipt part of the extension?

**Comment by davidanthoff** [comment]
Yes, the logic for this is [here](https://github.com/julia-vscode/julia-vscode/blob/693af39d974f204819ef9c9b6cc49f22e04dccf9/src/interactive/repl.ts#L86).

I don't see, though, how we could use a SHA here: the sys image doesn't store a SHA of the Manifest.toml that it was generated from, so we don't really have access to a SHA that we could compare the SHA of the Manifest.toml with...
